scripts/action-recognition/README.md

```diff
@@ -0,0 +1,65 @@
+# Action Recognition[1]
+[GluonCV Model Zoo](https://gluon-cv.mxnet.io/model_zoo/action_recognition.html)
+
+## Inference/Calibration Tutorial
+
+### FP32 inference
+
+```
+
+export CPUs=`lscpu | grep 'Core(s) per socket' | awk '{print $4}'`
+export OMP_NUM_THREADS=${CPUs}
+export KMP_AFFINITY=granularity=fine,noduplicates,compact,1,0
+
+# dummy data
+python test_recognizer.py --model inceptionv3_ucf101 --use-pretrained --use-tsn --mode hybrid --input-size 299 --new-height 340 --new-width 450 --num-segments 3 --batch-size 64 --benchmark
```

Comment:
`--use-tsn` is deprecated, please remove it. Right now, if you set `--num-segments` larger than 1, TSN mode will be automatically turned on.

Reply:
Done